packages/tslint-config/src/rules/utils/addImportToFile.ts

```diff
@@ -0,0 +1,50 @@
+import { Replacement } from "tslint";
+import * as utils from "tsutils";
+import * as ts from "typescript";
+
+export function addImportToFile(file: ts.SourceFile, imports: string[], packageName: string) {
+    const packageToModify = file.statements.find(
+        statement => utils.isImportDeclaration(statement) && statement.moduleSpecifier.getText() === `"${packageName}"`
+    ) as ts.ImportDeclaration;
+    if (
+        packageToModify &&
+        packageToModify.importClause &&
+        packageToModify.importClause.namedBindings &&
+        utils.isNamedImports(packageToModify.importClause.namedBindings)
+    ) {
+        const existingImports = packageToModify.importClause.namedBindings.elements.map(el => el.name.getText());
+        // Poor man's lodash.uniq without the dep.
+        const newImports = Array.from(new Set(existingImports.concat(imports))).sort((a, b) =>
+            a.toLowerCase().localeCompare(b.toLowerCase())
+        );
+        const importString = `{ ${newImports.join(", ")} }`;
+        return Replacement.replaceNode(packageToModify.importClause.namedBindings, importString);
+    } else {
+        // we always place the import in alphabetical order. If imports are already alpha-ordered, this will act nicely
+        // with existing lint rules. If imports are not alpha-ordered, this may appear weird.
+        const allImports = file.statements.filter(utils.isImportDeclaration);
+        const newImportIndex = allImports.findIndex(imp => {
+            // slice the quotes off each module specifier
+            return compare(imp.moduleSpecifier.getText().slice(1, -1), packageName) === 1;
+        });
+        const startIndex = newImportIndex === -1 ? 0 : allImports[newImportIndex].getStart();
+        return Replacement.appendText(startIndex, `import { ${imports.join(", ")} } from "${packageName}";\n`);
+    }
+}
+
+// taken from tslint orderedImportRules
+function compare(a: string, b: string): 0 | 1 | -1 {
+    function isLow(value: string) {
```

Comment:
pull this out to root scope to avoid redefining it each time `compare()` is called.